.github/workflows/deploy-infra.yaml

```diff
@@ -495,7 +495,9 @@ jobs:
               .myAccount.accountId = strenv(AWS_ACCOUNT_ID) |
               .myAccount.publicFacing = strenv(PUBLIC_FACING) |
               .myAccount.acmCertificate = strenv(ACM_CERTIFICATE_ARN) |
-              .myAccount.selfSignedCertificate = strenv(SELF_SIGNED_CERTIFICATE)
+              .myAccount.selfSignedCertificate = strenv(SELF_SIGNED_CERTIFICATE) |
+              .myAccount.workerTimeout = strenv(WORKER_TIMEOUT) |
+              .myAccount.workerScaleToZero = strenv(WORKER_SCALE_TO_ZERO)
```

Comment:
these don't seem to make it to `worker` config. Am I missing something or does this need to be merged to master for it to work?